### ISSUE TITLE
Cannot connect to Keycloak 20

### ISSUE THREAD

**Issue by thomasdarimont** [adorsys/keycloak-config-cli#805]
### Current Behavior

Keycloak 20 introduced new properties which are not yet supported by keycloak-config-cli
```
dev-acme-keycloak-provisioning-1  | javax.ws.rs.ProcessingException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "cryptoInfo" (class org.keycloak.representations.info.ServerInfoRepresentation), not marked as ignorable (14 known properties: "protocolMapperTypes", "providers", "identityProviders", "themes", "passwordPolicies", "clientInstallations", "memoryInfo", "enums", "socialProviders", "clientImporters", "profileInfo", "componentTypes", "systemInfo", "builtinProtocolMappers"])
dev-acme-keycloak-provisioning-1  |  at [Source: (org.jboss.resteasy.specimpl.AbstractBuiltResponse$InputStreamWrapper); line: 1, column: 1096] (through reference chain: org.keycloak.representations.info.ServerInfoRepresentation["cryptoInfo"])
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.client.jaxrs.internal.ClientResponse.readFrom(ClientResponse.java:254)
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.specimpl.BuiltResponse.readEntity(BuiltResponse.java:90)
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.specimpl.AbstractBuiltResponse.readEntity(AbstractBuiltResponse.java:256)
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.extractResult(ClientInvocation.java:167)
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.client.jaxrs.internal.proxy.extractors.BodyEntityExtractor.extractEntity(BodyEntityExtractor.java:64)
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientInvoker.invokeSync(ClientInvoker.java:154)
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientInvoker.invoke(ClientInvoker.java:115)
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientProxy.invoke(ClientProxy.java:76)
dev-acme-keycloak-provisioning-1  | 	at jdk.proxy2/jdk.proxy2.$Proxy78.getInfo(Unknown Source)
dev-acme-keycloak-provisioning-1  | 	at de.adorsys.keycloak.config.provider.KeycloakProvider.lambda$getKeycloakWithRetry$1(KeycloakProvider.java:131)
dev-acme-keycloak-provisioning-1  | 	at net.jodah.failsafe.Functions.lambda$get$0(Functions.java:48)
dev-acme-keycloak-provisioning-1  | 	at net.jodah.failsafe.RetryPolicyExecutor.lambda$supply$0(RetryPolicyExecutor.java:66)
dev-acme-keycloak-provisioning-1  | 	at net.jodah.failsafe.Execution.executeSync(Execution.java:128)
dev-acme-keycloak-provisioning-1  | 	at net.jodah.failsafe.FailsafeExecutor.call(FailsafeExecutor.java:379)
dev-acme-keycloak-provisioning-1  | 	at net.jodah.failsafe.FailsafeExecutor.get(FailsafeExecutor.java:68)
dev-acme-keycloak-provisioning-1  | 	at de.adorsys.keycloak.config.provider.KeycloakProvider.getKeycloakWithRetry(KeycloakProvider.java:129)
dev-acme-keycloak-provisioning-1  | 	at de.adorsys.keycloak.config.provider.KeycloakProvider.createKeycloak(KeycloakProvider.java:108)
dev-acme-keycloak-provisioning-1  | 	at de.adorsys.keycloak.config.provider.KeycloakProvider.getInstance(KeycloakProvider.java:76)
dev-acme-keycloak-provisioning-1  | 	at de.adorsys.keycloak.config.repository.RealmRepository.getResource(RealmRepository.java:55)
dev-acme-keycloak-provisioning-1  | 	at de.adorsys.keycloak.config.repository.RealmRepository.get(RealmRepository.java:59)
dev-acme-keycloak-provisioning-1  | 	at de.adorsys.keycloak.config.repository.RealmRepository.exists(RealmRepository.java:46)
dev-acme-keycloak-provisioning-1  | 	at de.adorsys.keycloak.config.service.RealmImportService.doImport(RealmImportService.java:128)
dev-acme-keycloak-provisioning-1  | 	at de.adorsys.keycloak.config.KeycloakConfigRunner.run(KeycloakConfigRunner.java:79)
dev-acme-keycloak-provisioning-1  | 	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:771)
dev-acme-keycloak-provisioning-1  | 	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:755)
dev-acme-keycloak-provisioning-1  | 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
dev-acme-keycloak-provisioning-1  | 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1306)
dev-acme-keycloak-provisioning-1  | 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1295)
dev-acme-keycloak-provisioning-1  | 	at de.adorsys.keycloak.config.KeycloakConfigApplication.main(KeycloakConfigApplication.java:35)
dev-acme-keycloak-provisioning-1  | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
dev-acme-keycloak-provisioning-1  | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
dev-acme-keycloak-provisioning-1  | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
dev-acme-keycloak-provisioning-1  | 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
dev-acme-keycloak-provisioning-1  | 	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
dev-acme-keycloak-provisioning-1  | 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
dev-acme-keycloak-provisioning-1  | 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
dev-acme-keycloak-provisioning-1  | 	at org.springframework.boot.loader.PropertiesLauncher.main(PropertiesLauncher.java:467)
dev-acme-keycloak-provisioning-1  | Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "cryptoInfo" (class org.keycloak.representations.info.ServerInfoRepresentation), not marked as ignorable (14 known properties: "protocolMapperTypes", "providers", "identityProviders", "themes", "passwordPolicies", "clientInstallations", "memoryInfo", "enums", "socialProviders", "clientImporters", "profileInfo", "componentTypes", "systemInfo", "builtinProtocolMappers"])
dev-acme-keycloak-provisioning-1  |  at [Source: (org.jboss.resteasy.specimpl.AbstractBuiltResponse$InputStreamWrapper); line: 1, column: 1096] (through reference chain: org.keycloak.representations.info.ServerInfoRepresentation["cryptoInfo"])
dev-acme-keycloak-provisioning-1  | 	at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
dev-acme-keycloak-provisioning-1  | 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1127)
dev-acme-keycloak-provisioning-1  | 	at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2023)
dev-acme-keycloak-provisioning-1  | 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1700)
dev-acme-keycloak-provisioning-1  | 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1678)
dev-acme-keycloak-provisioning-1  | 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:319)
dev-acme-keycloak-provisioning-1  | 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:176)
dev-acme-keycloak-provisioning-1  | 	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:323)
dev-acme-keycloak-provisioning-1  | 	at com.fasterxml.jackson.databind.ObjectReader._bind(ObjectReader.java:2025)
dev-acme-keycloak-provisioning-1  | 	at com.fasterxml.jackson.databind.ObjectReader.readValue(ObjectReader.java:1175)
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.plugins.providers.jackson.ResteasyJackson2Provider.readFrom(ResteasyJackson2Provider.java:193)
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.core.interception.jaxrs.AbstractReaderInterceptorContext.readFrom(AbstractReaderInterceptorContext.java:101)
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.core.interception.jaxrs.AbstractReaderInterceptorContext.proceed(AbstractReaderInterceptorContext.java:80)
dev-acme-keycloak-provisioning-1  | 	at org.jboss.resteasy.client.jaxrs.internal.ClientResponse.readFrom(ClientResponse.java:217)
dev-acme-keycloak-provisioning-1  | 	... 36 common frames omitted
```

### Steps To Reproduce

```markdown
1. Run keycloak config cli against Keycloak 20.0.0 with `LOGGING_LEVEL_ROOT: "DEBUG"`
2. Observe the exception above in the log
```

### Environment

- Keycloak Version: 20.0.0
- keycloak-config-cli Version: quay.io/adorsys/keycloak-config-cli:5.3.1-19.0.1
- Java Version: 17

**Comment by thomasdarimont** [comment]
See: #806

**Comment by LCerebo** [comment]
I can confirm that this pull request fix the problem with keycloak 20.0.x.
Tested on keycloak 20.0.0
